### PR TITLE
Link to archives of missing RHACM+SubM blogs

### LINKED ISSUE #1026: Broken link detected by CI
Periodic link aliveness CI detected a broken link. Please see the [periodic job
results](https://github.com/submariner-io/submariner-website/actions?query=workflow%3APeriodic) for details.

### PR DESCRIPTION
**Pull request by dfarrell07**
The broken link tests recently started failing on these two posts.

They also 404 for me. Other posts from the same blogger still work.

Fixes: #1026
Depends on https://github.com/submariner-io/submariner-website/pull/1032

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

File: src/content/other-resources/_index.en.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ This page catalogs content documenting Submariner elsewhere on the web.
...
 * [Set up Istio Multicluster with Submariner in Red Hat Advanced Cluster Management for Kubernetes (2021-10)](https://cloud.redhat.com/blog/set-up-istio-multicluster-with-submariner-in-red-hat-advanced-cluster-management-for-kubernetes)
 * [Multi-Cluster monitoring with Prometheus and Submariner (2021-09)](https://medium.com/@danielbachar/multi-cluster-monitoring-with-prometheus-and-submariner-f89ff733e7ec)
-* [Connecting stateful applications in multicluster environments with RHACM and Submariner (2021-04)](https://rcarrata.com/openshift/rhacm-submariner-2/)
-* [Deep Dive with RHACM and Submariner - Connecting multicluster overlay networks (2021-04)](https://rcarrata.com/openshift/rhacm-submariner/)
+* [Connecting stateful applications in multicluster environments with RHACM and Submariner (2021-04)](https://web.archive.org/web/20221130050657/https://rcarrata.com/openshift/rhacm-submariner-2/)
+* [Deep Dive with RHACM and Submariner - Connecting multicluster overlay networks (2021-04)](https://web.archive.org/web/20230531045006/https://rcarrata.com/openshift/rhacm-submariner/)
```
Comment (sridhargaddam) [inline src/content/other-resources/_index.en.md:45]: This seems to have moved here.
https://rcarrata.com/openshift/multicluster/networking/kubernetes/rhacm-submariner/

File: src/content/other-resources/_index.en.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ This page catalogs content documenting Submariner elsewhere on the web.
 * [Set up an Istio Multicluster Service Mesh with Submariner in Red Hat Advanced Cluster Management for Kubernetes (2022-02)](https://cloud.redhat.com/blog/set-up-an-istio-multicluster-service-mesh-with-submariner-in-red-hat-advanced-cluster-management-for-kubernetes)
 * [Set up Istio Multicluster with Submariner in Red Hat Advanced Cluster Management for Kubernetes (2021-10)](https://cloud.redhat.com/blog/set-up-istio-multicluster-with-submariner-in-red-hat-advanced-cluster-management-for-kubernetes)
 * [Multi-Cluster monitoring with Prometheus and Submariner (2021-09)](https://medium.com/@danielbachar/multi-cluster-monitoring-with-prometheus-and-submariner-f89ff733e7ec)
-* [Connecting stateful applications in multicluster environments with RHACM and Submariner (2021-04)](https://rcarrata.com/openshift/rhacm-submariner-2/)
-* [Deep Dive with RHACM and Submariner - Connecting multicluster overlay networks (2021-04)](https://rcarrata.com/openshift/rhacm-submariner/)
+* [Connecting stateful applications in multicluster environments with RHACM and Submariner (2021-04)](https://web.archive.org/web/20221130050657/https://rcarrata.com/openshift/rhacm-submariner-2/)
```
Comment (sridhargaddam) [inline src/content/other-resources/_index.en.md:44]: https://rcarrata.com/openshift/multicluster/networking/kubernetes/rhacm-submariner-2/

**Review by skitt — CHANGES_REQUESTED** [review]
See @sridhargaddam’s comments.

**Comment by dfarrell07** [comment]
It seems the website's reorg has been reverted. The old links work again, the ones @sridhargaddam suggested no longer work.

**Pull request closed without merging**